kt/api-generator/src/main/kotlin/godot/codegen/workarounds/SignalWorkarounds.kt

```diff
@@ -0,0 +1,14 @@
+package godot.codegen.workarounds
+
+import godot.codegen.models.enriched.EnrichedSignal
+
+@Deprecated(
+    "This should be deleted when godot api is clean",
+    replaceWith = ReplaceWith("EnrichedSignal.name")
+)
+fun EnrichedSignal.sanitizedName(englobingClassName: String) =
```

Comment:
Put also a Todo here with the Godot Issue

Reply:
Should be fixed